### ISSUE TITLE
Flash plugin not work

### ISSUE THREAD

**Issue by gaochengyidlmu** [berstend/puppeteer-extra#75]
The work environment as follow.
```json
    "puppeteer": "^2.0.0",
    "puppeteer-extra": "^2.1.3",
    "puppeteer-extra-plugin-flash": "^2.1.3",
    "puppeteer-extra-plugin-user-data-dir": "^2.1.2",
    "puppeteer-extra-plugin-user-preferences": "^2.1.2"

    "nodejs": "12.13.1",
    "system": "macOS Catalina 10.15.1"
```

I execute the example code, but the flash is also been blocked.

```js
const puppeteer = require('puppeteer-extra');
puppeteer.use(require('puppeteer-extra-plugin-flash')())
;(async () => {
  const browser = await puppeteer.launch({ headless: false });
  const page = await browser.newPage();
  await page.goto('http://ultrasounds.com', { waitUntil: 'domcontentloaded' });
})();
```
Thank you very much!

**Comment by berstend** [comment]
Hey there, are you instructing pptr to use Google Chrome? The built-in Chromium won't work :)

**Comment by berstend** [comment]
Have a look at this example:
https://github.com/berstend/puppeteer-extra/blob/18d44900804c7fc16b3f770881ad13c3b7399768/packages/puppeteer-extra-plugin-flash/example.js

**Comment by berstend** [comment]
Closing this for now, please ping if you still have issues. :)

**Comment by Niek** [comment]
Same problem here, sample code:
```javascript
'use strict'

const puppeteer = require('puppeteer-extra')

const userName = require('os').userInfo().username
const pluginVersion = '32.0.0.303'
const pluginPath = `
  /Users/${userName}/Library/Application Support/Google/Chrome/PepperFlash/${pluginVersion}/PepperFlashPlayer.plugin
`.trim()

puppeteer.use(
    require('puppeteer-extra-plugin-flash')({
        pluginPath, // this should not be necessary, but setting just in case
        pluginVersion // the result is the same with or without
    })
);

(async () => {
    const browser = await puppeteer.launch({ headless: false, executablePath: '/Applications/Google\ Chrome.app/Contents/MacOS/Google\ Chrome', })
    const page = await browser.newPage()
    await page.goto('http://ultrasounds.com', { waitUntil: 'domcontentloaded' })
})()
```

Result: 
<img width="428" alt="Screen Shot 2019-12-16 at 14 37 13" src="https://user-images.githubusercontent.com/213140/70911271-d3e8be80-2011-11ea-87f7-b9e98fe84f0c.png">

chrome://version shows the correct FLash plugin/version, so I think it's more of a preference issue.

**Comment by berstend** [comment]
Fix has been published as `puppeteer-extra-plugin-flash@2.2.5`
